PWGHF/HFL/Tasks/CMakeLists.txt

```diff
@@ -13,3 +13,8 @@ o2physics_add_dpl_workflow(task-single-muon
                     SOURCES taskSingleMuon.cxx
                     PUBLIC_LINK_LIBRARIES O2::Framework O2Physics::AnalysisCore O2::DetectorsBase O2Physics::AnalysisCore O2Physics::PWGDQCore
                     COMPONENT_NAME Analysis)
+
+o2physics_add_dpl_workflow(mu-charm-beauty-separation
+                    SOURCES taskHFmuCharmBeautySep.cxx
```

Comment:
The names still don't match. Look at the workflow above. I would suggest:
- file name: `taskMuonCharmBeautySeparation.cxx`
- workflow name: `task-muon-charm-beauty-separation`
- task name: `HfTaskMuonCharmBeautySeparation`

Reply:
See https://aliceo2group.github.io/analysis-framework/docs/advanced-specifics/pwghf.html#naming-conventions

Reply:
Thanks a lot for the comments.
I have changed them